app/services/live_stream_updater.rb

```diff
@@ -45,29 +44,30 @@ def publish_content_item
       begin
         Services.publishing_api.publish(landing_page_id, "minor")
       rescue GdsApi::HTTPErrorResponse
-        object.update(url: live_url)
```

Comment:
There was no point in this line, as the content item has already been updated. ie the object and the content item are in sync.